tests/aeadtest.sh

```diff
@@ -3,6 +3,8 @@ set -e
 TEST=./aeadtest
 if [ -e ./aeadtest.exe ]; then
 	TEST=./aeadtest.exe
+elif [ -e ./aeadtest.js ]; then
```

Comment:
don't forget to add/push aeadtest.js as well

Reply:
I don't think it should included.
aeadtest.js file is generated by emscripten when it compiles the aeadtest target to wasm